README.md

```diff
@@ -21,49 +21,167 @@
 
 ## About
 
-The CrateDB MCP Server is suitable for Text-to-SQL and documentation retrieval,
-specializing on the CrateDB database.
+The CrateDB MCP Server for natural-language Text-to-SQL and documentation
+retrieval specializes in CrateDB database clusters.
 
 The Model Context Protocol ([MCP]) is a protocol that standardizes providing
-context to LLMs like Claude, ChatGPT, and MistralAI.
+context to language models and AI assistants.
 
-## Features
+### Introduction
 
-The CrateDB MCP Server lets these LLMs operate directly on CrateDB, enabling
-use cases like:
+The CrateDB Model Context Protocol (MCP) Server connects AI assistants directly
+to your CrateDB clusters and the CrateDB knowledge base, enabling seamless
+interaction through natural language.
 
-- Answer questions about your data and database state.
-- Help you debug and optimize queries directly on the database, for tasks
-  like optimizing queries using CrateDB-specific capabilities and syntax.
-- Have general conversations about any details of CrateDB and CrateDB Cloud.
+It serves as a bridge between AI tools and your analytics database,
+allowing you to analyze data, the cluster state, troubleshoot issues, and
+perform operations using conversational prompts.
 
-To use an MCP server, you need a [client that supports] the protocol. The most
-notable ones are Claude Desktop, ChatGPT desktop, OpenAI agents SDK, and Cursor.
+**Experimental:** Please note that the CrateDB MCP Server is an experimental
+feature provided as-is without warranty or support guarantees. Enterprise
+customers should use this feature at their own discretion.
 
-### Details
+### Quickstart Guide
 
-The application includes two independent subsystems: The Text-to-SQL API talks
-to a CrateDB database cluster, while the documentation server looks up guidelines
-specific to CrateDB topics based on user input on demand, for example, from
-<https://cratedb.com/docs>, to provide the most accurate possible information.
-Relevant information is relayed per [cratedb-outline.yaml].
+The CrateDB MCP Server is compatible with AI assistants that support the Model
+Context Protocol (MCP), either using standard input/output (stdio),
+server-sent events (SSE), or HTTP Streams (streamable-http).
 
-- Database / Text-to-SQL: `get_health`, `get_table_metadata`, `query_sql`
-- Documentation server: `get_cratedb_documentation_index`, `fetch_cratedb_docs`
+To use the MCP server, you need a [client that supports][MCP clients] the
+protocol. The most notable ones are ChatGPT, Claude, Cline Bot, Cursor,
+GitHub Copilot, Mistral AI, OpenAI Agents SDK, Windsurf, and others.
 
-### Examples
+The `uvx` launcher command is provided by the [uv] package manager.
+The [installation docs](#install) section includes guidelines on how to
+install it on your machine.
```

Comment:
Excerpt from the backlog, about getting rid of `uvx`?

https://github.com/crate/cratedb-mcp/blob/89a4dc07e8dd1a6a8841b8f801582034db164b0c/docs/backlog.md?plain=1#L14-L16

Reply:
I took a note about it.

- GH-27